### PR TITLE
Add working copy support to weblinks FACE UI

### PR DESCRIPTION
**Pull request by CodeBaboon**
## Description
​
Working copy actions (checkout/commit/cancel) have been added to the weblinks domain siren entities via https://github.com/Brightspace/lms/pull/6280. We need to make use of these actions within the activities UI.
​
## Goal/Solution
​
This PR adds basic support for calling checkout and commit when working copy actions are found on the provided weblinks entity.

## Video/Screenshots
​
N/A

​
## Remaining Work
- [x] This PR relies on siren-sdk changes found in https://github.com/BrightspaceHypermediaComponents/siren-sdk/pull/285. Tests will fail until the siren-sdk PR is merged and this PR is updated to use the new release of siren-sdk.
- [x] With the [removal of the cancel action](https://github.com/Brightspace/lms/pull/6280/commits/4d02f64f7b7410d1444a752744b5e1b70b10c884) we need to remove [this line](https://github.com/BrightspaceHypermediaComponents/activities/pull/1584/files#diff-8748b015ce688539098657d43c898e19c2c6309477095e9aa7505a9d6666071aR18).
​​

## Relevant Rally Links 
​
[US124553](https://rally1.rallydev.com/#/289692574792d/custom/355050439968?detail=%2Fuserstory%2F490369835660&fdp=true?fdp=true): Weblinks > Use working copy during FACE create workflow

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-content-editor/web-link/d2l-activity-content-web-link-detail.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ class ContentWebLinkDetail extends AsyncContainerMixin(SkeletonMixin(ErrorHandli
 		this._debounceJobs = {};
 		this._setEntityType(ContentWebLinkEntity);
 		this.skeleton = true;
-		this.saveOrder = 2000;
+		this.saveOrder = 500;
```
Comment (CodeBaboon) [inline components/d2l-activity-editor/d2l-activity-content-editor/web-link/d2l-activity-content-web-link-detail.js:33]: This is necessary in order to make sure that created-in-working-copy-land weblinks are persisted to the datastore ahead of outside-of-working-copy actions like availability dates are saved.

File: components/d2l-activity-editor/d2l-activity-content-editor/web-link/state/content-web-link.js [hunk 2]
```diff
@@ -25,7 +28,8 @@ export class ContentWebLink {
 	async fetch() {
 		const sirenEntity = await fetchEntity(this.href, this.token);
 		if (sirenEntity) {
-			const entity = new ContentWebLinkEntity(sirenEntity, this.token, { remove: () => { } });
+			let entity = new ContentWebLinkEntity(sirenEntity, this.token, { remove: () => { } });
+			entity = await this._checkout(entity);
```
Comment (ianlaidlaw) [inline components/d2l-activity-editor/d2l-activity-content-editor/web-link/state/content-web-link.js:32]: what happens here if working copy isn't turned on and this returns `undefined` ? is it okay that we're overriding the entity and then calling `load` ?

Comment (ianlaidlaw) [reply components/d2l-activity-editor/d2l-activity-content-editor/web-link/state/content-web-link.js:32]: nevermind! I see that you return the same entity. I was looking at the `if (!webLinkEntity) {`

**Review by ianlaidlaw — DISMISSED** [review]
LGTMEEEE

File: components/d2l-activity-editor/d2l-activity-content-editor/web-link/d2l-activity-content-web-link-detail.js (hunk 1 above)
Comment (brissons) [reply components/d2l-activity-editor/d2l-activity-content-editor/web-link/d2l-activity-content-web-link-detail.js:33]: Just checking, so for the other working copy-ized stuff down the road, we should also set those to 500?

**Review by brissons — APPROVED** [review]
lgtm!

File: components/d2l-activity-editor/d2l-activity-content-editor/web-link/d2l-activity-content-web-link-detail.js (hunk 1 above)
Comment (CodeBaboon) [reply components/d2l-activity-editor/d2l-activity-content-editor/web-link/d2l-activity-content-web-link-detail.js:33]: Yes, or at least something lower than the availability dates stuff (which is `1000` I believe).

Comment (brissons) [reply components/d2l-activity-editor/d2l-activity-content-editor/web-link/d2l-activity-content-web-link-detail.js:33]: Thanks!

**Review by brissons — APPROVED** [review]

**Pull request merged**